install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -85,6 +85,24 @@ data:
     outboundTrafficPolicy:
       mode: {{ .Values.global.outboundTrafficPolicy.mode }}
 
+    {{- if .Values.global.configRootNamespace }}
```

Comment:
Clamping down in global - is this really a 'global' ? Why ? Only pilot needs it, can be Values.configRootNamespace

( we need to clenup the globals - but we can start by not adding more )

Reply:
please point me to an example that I can follow. Happy to fix

Reply:
this has to be a global as there are destination rules in different subsystems that apply cluster wide.